front/src/assets/variables.tsx

```diff
@@ -0,0 +1,45 @@
+type Media = {
+  xs: string;
+  sm: string;
+  md: string;
+  lg: string;
+  xl: string;
+  xxl: string;
+};
+
+export const media: Media = {
+  xs: "@media (max-width: 480px)",
+  sm: "@media (max-width: 576px)",
+  md: "@media (max-width: 768px)",
+  lg: "@media (max-width: 992px)",
+  xl: "@media (max-width: 1200px)",
+  xxl: "@media (max-width: 1600px)",
+};
+
+type ThemeCss = {
+  xs_text: string;
+  md_text: string;
+};
+
+export const themeCss: ThemeCss = {
+  xs_text: "10px",
+  md_text: "12px",
+};
+
+type Css = {
+  xs_text: string;
+  md_text: string;
+};
+
+export const css: Css = {
```

Comment:
zagmatwane to mocno. Ja mam z reguly tak:

export type FontSize =
  | 'xxs'
  | 'xs'
  | 'sm'
  | 'base'
  | 'md'
  | 'xl'
  | 'xxl'
  | 'xxxl'
export type FontWeight =
  | 'light'
  | 'normal'
  | 'medium'
  | 'semiBold'
  | 'bold'
  | 'black'

export type Font = {
  size: Record<FontSize, string>
  weight: Record<FontWeight, number>
}

export const font: Font = {
  size: {
    xxs: '0.6875rem',
    xs: '0.75rem',
    sm: '0.8125rem',
    base: '1rem',
    md: '1.25rem',
    xl: '1.375rem',
    xxl: '1.5rem',
    xxxl: '1.625rem',
  },
  weight: {
    light: 300,
    normal: 400,
    medium: 500,
    semiBold: 600,
    bold: 700,
    black: 900,
  },
}

Reply:
skipped.

Reply:
Done.